docs/src/main/asciidoc/security-oidc-bearer-token-authentication-tutorial.adoc

```diff
@@ -228,13 +228,13 @@ docker run --name keycloak -e KC_BOOTSTRAP_ADMIN_USERNAME=admin -e KC_BOOTSTRAP_
 
 For more information, see the Keycloak documentation about link:https://www.keycloak.org/docs/latest/server_admin/index.html#configuring-realms[creating and configuring a new realm].
 
-ifndef::no-quarkus-keycloak-admin-client[]
+ifndef::no-quarkus-keycloak-admin-resteasy-client[]
 [NOTE]
 ====
 If you want to use the Keycloak Admin Client to configure your server from your application, you need to include either the `quarkus-keycloak-admin-rest-client` or the `quarkus-keycloak-admin-resteasy-client` (if the application uses `quarkus-rest-client`) extension.
```

Comment:
Not related to this PR but should be fixed,

```suggestion
If you want to use the Keycloak Admin Client to configure the Keycloak server from your application, you need to include either the `quarkus-keycloak-admin-rest-client` or the `quarkus-keycloak-admin-resteasy-client` extension.
```

The former should be chosen if the `quarkus-rest-client` is used and the latter if the  `quarkus-resteasy-client` is used, but I'm not sure we have clarify it in this note, if you prefer, please update this suggestion accordingly

Reply:
Important details to include. Thanks, @sberyozkin. Updated.